packages/generate/src/edgeql-js/generateOperatorTypes.ts

```diff
@@ -300,23 +300,23 @@ export function generateOperators({
     if (typeof args[0] === "string" && overloadDefs.Prefix[args[0]]) {
       op = args[0];
       params = [args[1]];
-      defs = overloadDefs.Prefix[op];
+      defs = overloadDefs.Prefix[op]!;
```

Comment:
I think it's more correct to add `| undefined` to the type of `defs`, instead of adding a non-null assertion here, since `op` may not be a valid name and there's a check for no valid defs being found below.